lib/generators/boring/devise/jwt/install/install_generator.rb

```diff
@@ -0,0 +1,224 @@
+# frozen_string_literal: true
+
+module Boring
+  module Devise
+    module Jwt
+      class InstallGenerator < Rails::Generators::Base
+        desc "Add devise-jwt to the application"
+
+        class_option :model_name, type: :string, aliases: "-m",
+                     default: "User",
+                     desc: "Tell us the user model name which will be used for authentication. Defaults to User"
+        class_option :use_env_variable, type: :boolean, aliases: "-ev",
+                     desc: "Use ENV variable for devise_jwt_secret_key. By default Rails credentials will be used."
+        class_option :revocation_strategy, type: :string, aliases: "-rs",
+                     enum: %w[JTIMatcher Denylist Allowlist],
+                     default: "Denylist",
+                     desc: "Tell us the revocation strategy to be used. Defaults to Denylist"
+        class_option :expiration_time_in_days, type: :numeric, aliases: "-et",
+                      default: 15,    
+                      desc: "Tell us the expiration time on days for the JWT token. Defaults to 15 days"
+
+        def verify_presence_of_devise_gem
+          gem_file_content_array = File.readlines("Gemfile")
+          devise_is_installed = gem_file_content_array.any? { |line| line.include?('devise') }
```

Comment:
I think I recently added a helper method called `gem_installed?`. can you try using that here?